runtime/default.policy

```diff
@@ -102,5 +102,5 @@ rules:
       open.filename =~ "/usr/bin/*" || 
       open.filename =~ "/usr/sbin/*" || 
       open.filename =~ "/opt/*") && 
-      open.flags & (O_RDWR | O_WRONLY) > 0 &&
-      process.name not in ["agent", "security-agent", "system-probe", "process-agent"]
+      open.flags & (O_CREAT | O_RDWR | O_WRONLY) > 0 &&
+      process.name not in ["agent", "security-agent", "system-probe", "process-agent", "postgres"]
```

Comment:
Could we maybe create 2 isolated rules:
- one with `/bin/*`, `/sbin/*`, `/usr/bin/*` ...
- another one with `/opt/*` only

I bet we only need to allow specific processes because of the `/opt/*` part. I don't see why any of those processes would need to modify something in one of the binary directories, and it would be best to avoid it.